packages/diagrams/backend/sirius-components-diagrams-layout/src/test/java/org/eclipse/sirius/components/diagrams/layout/services/DefaultTestDiagramDescriptionProvider.java

```diff
@@ -370,7 +374,7 @@ private EdgeDescription getDefaultEdgeDescription(Diagram diagram, LabelDescript
 
     }
 
-    private NodeDescription getDefaultNodeDescription(LabelDescription labelDescription, NodeDescription childDescription) {
+    private NodeDescription getDefaultNodeDescription(LabelDescription labelDescription, NodeDescription childDescription, boolean isCollapsible) {
```

Comment:
No need to change that, simply clone the NodeDescription for your test, it's do this way on purpose

Reply:
Technically a piece was missing that I'll add to make such use cases more easy to code. It was only working on Node/Diagram and not on NodeDescription/DiagramDescription